plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -659,24 +662,63 @@ class read_only : public api_base {
             walk_table_row_range( lower, upper );
          }
       }
-      return result;
+
+      // not enforcing the deadline for that second processing part (the serialization), as it is not taking place
+      // on the main thread, but in the http thread pool.
```

Comment:
I think this should be mentioned in the PR description.

Reply:
Oh yes will do!